app/scripts/lib/ppom/ppom-util.test.ts

```diff
@@ -1,61 +1,284 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Could we either remove this line and use `eslint-disable-next-line` throughout, or remove others and stick with this one?

Reply:
Good spot, will tidy this up in a future PR.